<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHRepeatedTest</class>
 <widget class="QDockWidget" name="QHRepeatedTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::NoDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Repeated Test</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shortcut:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shortcutcomboBox">
        <item>
         <property name="text">
          <string>Button</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">E</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">H</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">J</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">L</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">N</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">O</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Q</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">R</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">T</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">V</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">W</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Z</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Repeated Times:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timesspinBox">
        <property name="suffix">
         <string notr="true"/>
        </property>
        <property name="prefix">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="testpushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TEST</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stoppushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="infolabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
